.speakeasy/workflow.yaml

```diff
@@ -1,5 +1,5 @@
 workflowVersion: 1.0.0
-speakeasyVersion: v1.245.0
```

Comment:
This may have been discussed in the RFC but why do we prefer to have this here vs. just seeing what's in the gen.lock